src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -459,6 +459,9 @@ See the accompanying license.txt file for applicable licenses.
 
     <xsl:template name="processTopicNotices">
         <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
+            <xsl:if test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">
```

Comment:
The problem with fixing the issue in `processTopicNotices` template instead of the  `page-sequence.notice` attribute set that if someone wants to override that, they actually have to override the whole template, instead of just adding an attribute set. So my preference is using the attribute set in this case.